mcs/class/corlib/Test/System.IO/FileTest.cs

```diff
@@ -1317,6 +1317,10 @@ public void LastWriteTimeSubMs ()
 		[Test]
 		public void LastWriteTimeSubMsCopy ()
 		{
+#if MONOTOUCH
```

Comment:
Is the `#if` needed? Nice ideal is single IL for all platforms, even if larger.
Problem would be if env{SIMULATE_RUNTIME_VERSION} could mean something on other platforms.
Check the platform at runtime?

Reply:
it only applies to iOS (== MONOTOUCH) and we've been using that define in like >100 places across the BCL so "single IL" isn't a concern :)